opds.py

```diff
@@ -506,6 +506,10 @@ def groups(cls, _db, title, url, lane, annotator,
             return cached
 
         feed_size = Configuration.featured_lane_size()
+        if ignore_feed_size:
+            # This groups feed should return groups no matter how many
+            # books are found.
+            feed_size = 1
```

Comment:
We're chatting about this elsewhere, but note to self: the way that `feed_size` is being used here actually limits the number of featured works that are shown to 1, in addition to allowing them to be shown in groups at all. Definitely better to address this down below. Thanks, @leonardr.